Makefile

```diff
@@ -87,9 +76,9 @@ fmt:
 lint: golangci-lint fmt
 	@golangci-lint run -c golangci.yaml
 
-image: imagebuilder
-	@if [ $${USE_IMAGE_STREAM:-false} = false ] && [ $${SKIP_BUILD:-false} = false ] ; \
-	then hack/build-image.sh $(IMAGE_TAG) $(IMAGE_BUILDER) $(IMAGE_BUILDER_OPTS) ; \
+image:
+	@if [ $${SKIP_BUILD:-false} = false ] ; then \
+		buildah bud -t $(IMAGE_TAG) . ; \
```

Comment:
should we have a target for local dev where someone may not have `buildah` already on their system? or just let them resolve it separately (or add to docs?)

Reply:
IMO we should just document... you need the following tools to hack: x, y, z.